version.py

```diff
@@ -211,11 +211,13 @@ def process_version(
     return processed_version
 
 
-def find_latest_version(versions) -> str:
+def find_latest_version(versions, ignore_dev_versions=False) -> str:
```

Comment:
I think we should rather parse to semver (there is a util function for that in version.py), and check the parsing result for being "final"

Reply:
but we should still have the option to configure if dev versions should be ignored?

Reply:
should it be also possible to ignore dev versions in the other helper function like `find_latest_version_with_matching_minor` ?